python/paddle/fluid/tests/unittests/test_parallel_executor.py

```diff
@@ -565,24 +568,31 @@ def test_parallel_testing(self):
 depth = 8
 mix_hidden_lr = 1e-3
 embedding_name = 'emb'
+is_sparse = True
+use_nccl_allreduce = False
```

Comment:
Does it cover is_sparse=False and use_nccl_allreduce=True?

Reply:
Thanks, I will add unit test.